content/ja/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -92,7 +92,7 @@ Deploymentによって作成されたReplicaSetを管理しないでください
   コマンドの実行結果は以下のとおりです。
   ```shell
   Waiting for rollout to finish: 2 out of 3 new replicas have been updated...
-  deployment.apps/nginx-deployment successfully rolled out
+  deployment "nginx-deployment" successfully rolled out
```

Comment:
See my more detailed comment.  the changes to the content/ja file should be placed in its own PR.